src/promptflow/promptflow/_sdk/entities/_connection.py

```diff
@@ -437,6 +438,22 @@ def get_token(self):
 
         return self._token_provider.get_token()
 
+    @classmethod
+    def from_env(cls, name="default_env_connection"):
```

Comment:
shall this name default to be None ?

Reply:
Not sure, as this will be tracked in tracing feature.